csvqb/csvqb/configloaders/infojson.py

```diff
@@ -74,21 +75,22 @@ def _from_info_json_dict(d: Dict, data: pd.DataFrame):
     return Cube(metadata, data, columns)
 
 
-def _metadata_from_dict(config: dict) -> "CubeMetadata":
-    publisher = get_with_func_or_none(config, "publisher", URI)
-    return CubeMetadata(
+def _metadata_from_dict(config: dict) -> "CatalogMetadata":
+    publisher = get_with_func_or_none(config, "publisher", lambda p: str(GOV[uri_safe(p)]))
```

Comment:
Note that this does restrict the info.json just to representing Government publishers, however this is what gss-utils currently does, and I don't anticipate any external user wanting to use this info.json structure once we've put more thought into the external declarative file format we want.

Reply:
I don't think we're going to rip it out. This is the loader for the *historic* info.json syntax. AFAIK, we plan to create a new declarative JSON/YAML/Something syntax in the future; this is the one which should work for external users. The info.json is only ever going to be for us if you ask me.